policy/pipeline/artifacthub-pkg.yml

```diff
@@ -0,0 +1,19 @@
+# from https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml
+version: 0.0.1
+name: OPA Pipeline Policies
+displayName: OPA pipeline Policies
+createdAt: 2023-01-05T21:27:55.549418+00:00
+description: OPA policies enforcing standards on a tekton pipeline definition
+readme: |
+  OPA policies enforcing standards on a tekton pipeline definition
+install: |
+  `conftest pull git::https://github.com/hacbs-contract/ec-policies.git//policy`
```

Comment:
I think we want to use an oci:: url here, since that's the actual artifact.

Reply:
Also I think the git url should end in `//policy/pipeline` if we do want to use a git url.

Reply:
you're right, I did update it to use the oci url though